bundles/scripting/sightly/js-use-provider/pom.xml

```diff
@@ -70,6 +70,7 @@
                     <instructions>
                         <Sling-Bundle-Resources>/libs/sling/sightly/js;path:=/SLING-INF/libs/sling/sightly/js</Sling-Bundle-Resources>
                         <Require-Capability>io.sightly; filter:="(&amp;(version&gt;=1.0)(!(version&gt;=2.0)))"</Require-Capability>
+                        <Import-Package>org.mozilla.javascript;version="[1.1,2)",*</Import-Package>
```

Comment:
The more I look at this issue, the more I tend to think that the `org.mozilla.javascript;version=1.1.0` export from `org.apache.sling.scripting.javascript` bundle was wrong...

@kwin, @vladbailescu, should we actually define the range as `[1.7, 2)`, given that both  `org.apache.sling.scripting.javascript` and the JS Use Provider relied on Rhino 1.7?